src/components/color_picker/color_picker.ts

```diff
@@ -276,4 +276,8 @@ export class ColorPicker extends Component<ColorPickerProps, SpreadsheetChildEnv
       display === "block" ? `background-color:${background};left:${left};top:${top};` : ""
     }`;
   }
+
+  sameColor(color1: Color, color2: Color): boolean {
+    return isColorValid(color1) && isColorValid(color2) && toHex(color1) === toHex(color2);
+  }
```

Comment:
If you're motivated, we could change the tests that compare colors to use this. Or define a curstom jest matcher `isEqualColor`

Reply:
I'll do that !  Though I might miss some occurences in the tests